### ISSUE TITLE
NVDA ignores form field labels when <label> contains non-phrasing content

### ISSUE THREAD

**Issue by pbossley** [nvaccess/nvda#5777]
When a form field label contains a table in it, NVDA reports that there is no label to the field.
Although we do understand that this is not valid HTML e.g. <label> is phrasing content that should not contain a table, we would recommend / request that NVDA still read the label text as this is in use in some learning management systems.
See example:

`<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <title>Will Labels in Tables Report</title>
    </head>
    <body>
        <form action="#">
            <fieldset>
                <legend>Question One (Tables)</legend>
                <input type="radio" name="name1" value="name1" id="q11">
                <label for="q11"> <table> <tr> <td>Answer</td> </tr> <tr> <td>One</td> </tr> <tr> <td>for Question One</td> </tr> </table></label>
                <input type="radio" name="name1" value="name2" id="q12">
                <label for="q12"> <table> <tr> <td>Answer</td> </tr> <tr> <td>Two</td> </tr> <tr> <td>for Question One</td> </tr> </table></label>
                <input type="radio" name="name1" value="name3" id="q13">
                <label for="q13"> <table> <tr> <td>Answer</td> </tr> <tr> <td>Three</td> </tr> <tr> <td>for Question One</td> </tr> </table></label>
            </fieldset>
            <fieldset>
                <legend>Question Two (Just Phrasing Content)</legend>
                <input type="radio" name="name2" value="name4" id="q14">
                <label for="q14"><span>Answer <strong>One</strong> for Question Two</span></label>
                <input type="radio" name="name2" value="name5" id="q15">
                <label for="q15"><span>Answer <strong>Two</strong> for Question Two</span></label>
                <input type="radio" name="name2" value="name6" id="q16">
                <label for="q16"><span>Answer <strong>Three</strong> for Question Two</span></label>
            </fieldset>
            <input type="submit" name="sub" value="Submit">
        </form>

    </body>
</html>
`

**Comment by pbossley** [comment]
[radio-group-and-checkboxes-labels-in-tables.zip](https://github.com/nvaccess/nvda/files/146979/radio-group-and-checkboxes-labels-in-tables.zip)

**Comment by jcsteh** [comment]
When reporting focus, we use the label as calculated by the browser. Thus, you'd need to convince the various browser vendors to make this change. I'd say this is unlikely given that this probably isn't valid HTML, but that decision is not mine to make.

Note that in browse mode, if you cursor below the radio buttons, you can definitely see the text.

Note that you may also be able to work around this by using aria-labelledby and referencing the appropriate pieces of text.
